Checks/PWR022/example.c

```diff
@@ -1,13 +1,14 @@
 // PWR022: move invariant conditional out of the loop to facilitate
 // vectorization
 
-int example(int *A, int n) {
-  int total = 0;
+void example(int n, const double *A, const double *B, const double *C,
+             double *D) {
   for (int i = 0; i < n; i++) {
-    if (n < 10) {
-      total++;
+    for (int j = 0; j < n; j++) {
```

Comment:
nit: Use `++<var>` instead of `<var>++`. Just a [good practice](https://llvm.org/docs/CodingStandards.html#prefer-preincrement).

I know it won't affect this particular code, but we learn from what we read.